examples/sam/java/deploy.sh

```diff
@@ -9,11 +9,11 @@ sam build --use-container
 
 bucket="newrelic-example-${region}-${accountId}"
 
-aws s3 mb --region ${region} s3://${bucket}
+#aws s3 mb --region ${region} s3://${bucket}
 
-sam package --region ${region} --s3-bucket=${bucket} --output-template-file packaged.yaml
-aws cloudformation deploy --region ${region} \
+sam package --region ${region} --s3-bucket ${bucket} --output-template-file packaged.yaml
+sam deploy --region ${region} \
   --template-file packaged.yaml \
-  --stack-name NewrelicExampleJava \
+  --stack-name NewrelicExampleJava2 \
   --capabilities CAPABILITY_IAM \
```

Comment:
Did you mean to commit these?

Reply:
@MattWhelan No, I thought I had reset those. Will revise it. Was only supposed to update to latest version.